src/lib/Menu/MainMenuBuilder.php

```diff
@@ -247,7 +249,7 @@ public function createStructure(array $options): ItemInterface
         $contentMenuItems = $this->getContentMenuItems();
         $adminMenuItems = $this->getAdminMenuItems();
 
-        $menu[self::ITEM_CONTENT]->setChildren($contentMenuItems);
+        array_map([$menu[self::ITEM_CONTENT], 'addChild'], $contentMenuItems);
```

Comment:
Personally I'd pass the content menu object into `getContentMenuItems()` method, and `addChild` there. But that's my personal preference.